index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>calculator</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="screen" value="" disabled />
+    </div>
+
+    <button id="clearButton">AC</button>
+    <button class="operationButtons">/</button>
+
+    <br />
+    <button numbersToPress>1</button>
+    <button numbersToPress>2</button>
+    <button numbersToPress>3</button>
+    <button class="operationButtons">*</button>
+    <br />
+    <button numbersToPress>4</button>
+    <button numbersToPress>5</button>
+    <button numbersToPress>6</button>
+    <button class="operationButtons">+</button>
+    <br />
+    <button numbersToPress>7</button>
+    <button numbersToPress>8</button>
+    <button numbersToPress>9</button>
+    <button class="operationButtons">-</button>
+    <br />
+    <button numbersToPress>0</button>
+    <button id="equalsButton">=</button>
+    <button id = "decimalButton">.</button>
+    <button class="operationButtons">^</button>
+    <script src="calculator.js"></script>
```

Comment:
this file tries to set module.exports = Calculator. So above this line please put the following code:
```
    <script>
      const module = {};
    </script>
```

Reply:
That should stop an error from happening

Reply:
Ok, thanks for that.